src/components/mypage/Editor/Editor.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import * as S from './Editor.Styled';
+
+interface EditorProps {
+  title: string;
+  onClose: () => void;
+  children: React.ReactNode;
+}
+
+const Editor: React.FC<EditorProps> = ({ title, onClose, children }) => (
+  <S.EditorContainer>
```

Comment:
```suggestion
  <S.EditorContainer ref={editorRef}>
```
이 컨테이너를 외부 클릭 감지 기준 DOM 요소로 설정하는게 좋을 것 같습니당